Traceback (most recent call last):
  File "/home/mudit/bot-battle/submission4/submission.py", line 397, in <module>
    main()
  File "/home/mudit/bot-battle/submission4/submission.py", line 67, in main
    query = game.get_next_query()
  File "/home/mudit/bot-battle/risk-helper/src/risk_helper/game.py", line 36, in get_next_query
    query = self.connection.get_next_query()
  File "/home/mudit/bot-battle/risk-helper/src/risk_helper/connection.py", line 51, in get_next_query
    return DiscriminatedTypeAdapter.model_validate_json(self._receive()).root
  File "/home/mudit/bot-battle/risk-helper/src/risk_helper/connection.py", line 31, in _receive
    buffer.extend(self._from_engine_pipe.read(1).encode())
KeyboardInterrupt
